src/main/java/org/broadinstitute/consent/http/db/StudyDAO.java

```diff
@@ -0,0 +1,89 @@
+package org.broadinstitute.consent.http.db;
+
+import org.broadinstitute.consent.http.db.mapper.FileStorageObjectMapperWithFSOPrefix;
+import org.broadinstitute.consent.http.db.mapper.StudyReducer;
+import org.broadinstitute.consent.http.models.FileStorageObject;
+import org.broadinstitute.consent.http.models.Study;
+import org.jdbi.v3.sqlobject.config.RegisterBeanMapper;
+import org.jdbi.v3.sqlobject.config.RegisterRowMapper;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.statement.GetGeneratedKeys;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+import org.jdbi.v3.sqlobject.statement.UseRowReducer;
+import org.jdbi.v3.sqlobject.transaction.Transactional;
+
+import java.time.Instant;
+import java.util.List;
+import java.util.UUID;
+
+@RegisterBeanMapper(Study.class)
+@RegisterRowMapper(FileStorageObjectMapperWithFSOPrefix.class)
+public interface StudyDAO extends Transactional<StudyDAO> {
+
+
+    @UseRowReducer(StudyReducer.class)
+    @SqlQuery("""
+        SELECT
+            s.*,
+            sp.study_property_id AS sp_study_property_id,
+            sp.study_id AS sp_study_id,
+            sp.key AS sp_key,
+            sp.value AS sp_value,
+            sp.type AS sp_type,
+            d.dataset_id AS s_dataset_id,
+        """
+        + FileStorageObject.QUERY_FIELDS_WITH_FSO_PREFIX + " " +
+        """
```

Comment:
I'm surprised this hasn't triggered a semgrep error since it looks for this kind of pattern for sql injection problems.